lib/api/v3/utilities/custom_field_injector.rb

```diff
@@ -302,6 +307,19 @@ def property_value_setter_for(custom_field)
           }
         end
 
+        def calculated_value_error_getter(custom_field)
+          ->(*) {
+            next unless custom_field.calculated_value?
+            next unless available_custom_fields.include?(custom_field)
+
+            errors = custom_field.calculated_value_errors.where(customized_id: id, customized_type: "Project")
+            errors.map do |err|
+              { error_code: err.error_code,
+                error_message: CalculatedValues::ErrorsHelper.calculated_value_error_msg(err) }
```

Comment:
Should it have more compact keys:
```suggestion
              { code: err.error_code,
                message: CalculatedValues::ErrorsHelper.calculated_value_error_msg(err) }
```